tools/cloud-build/images/test-runner/terraformrc

```diff
@@ -0,0 +1,12 @@
+# .terraformrc setup for google network mirror
+provider_installation {
+  network_mirror {
+    url = "https://storage.googleapis.com/network-mirror-private-provider/"
+    include = ["hashicorp/google-private"]
+    exclude = ["hashicorp/null", "hashicorp/local", "hashicorp/random", "hashicorp/google", "hashicorp/google-beta"]
```

Comment:
What is motivation to use both `include` and `exclude` here?

Reply:
Just to be extra careful about what we're pulling from the network mirror.